plenum/test/replica_removing/test_replica_removing.py

```diff
@@ -102,22 +102,31 @@ def test_ordered_request_freed_on_replica_removal(looper,
                                                   chkFreqPatched,
                                                   view_change):
     node = txnPoolNodeSet[0]
-    sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle, sdk_wallet_client, 1)
+    if node.master_replica._consensus_data.checkpoints:
+        num_requests_to_stable_checkpoint = node.master_replica._consensus_data.stable_checkpoint - \
+                                            node.master_replica._consensus_data.checkpoints[-1].seqNoEnd
+        sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle, sdk_wallet_client,
+                                  num_requests_to_stable_checkpoint)
+    assert not node.master_replica._consensus_data.checkpoints
```

Comment:
We should always have at least 1 Checkpoint (at least after https://github.com/hyperledger/indy-plenum/pull/1317)

Reply:
Yes, but at the moment in this case the list is empty, and we can't use data from it.